examples/trials/benchmarking/automlbenchmark/nni/extensions/NNI/architectures/run_random_forest.py

```diff
@@ -146,14 +146,14 @@ def run_random_forest(dataset, config, tuner, log):
             elif limit_type == 'ntrials':
                 if trial_count >= trial_limit:
                     break
-        except:
+        except Exception as err:
+            log.info(err)
```

Comment:
log.warning? log.warn? Have you tested this?

Reply:
sorry,tht file was i fixing before, wrong commitsof this file